### PR TITLE
V1 release

### PR DESCRIPTION
**Pull request by Sqyphen**
A branch to hi-light the initial showing of the project Barcode code base.
Please review and question 😄

### CODE REVIEW

File: routes/barcode.js [hunk 1]
```diff
@@ -19,14 +19,15 @@ router.get('/', async (req, res, next) => {
     const images = await article.getImagesFromDateRange(dateFrom, dateTo);
     const config = createConfig(orientation, fit, images.length, width, height);
     const updatedImages = createImages(images, fit, config.width, config.height);
-    const outputPath = './downloads/_result/output.jpg';
+    const downloadPath = process.env.DOWNLOAD_PATH;
```
Comment (Lily2point0) [inline routes/barcode.js]: I would use the `path` module to ensure your paths are all relative to the root

File: routes/barcode.js [hunk 2]
```diff
@@ -19,14 +19,15 @@ router.get('/', async (req, res, next) => {
...
+    const outputPath = `${downloadPath}_result/output.jpg`;
```
Comment (Lily2point0) [inline routes/barcode.js]: check existence of `_results` with `fs`

Comment (Sqyphen) [reply routes/barcode.js]: Yes, added this in, thanks

File: routes/barcode.js (hunk 1 above)
Comment (Sqyphen) [reply routes/barcode.js]: Done! :)

**Pull request merged**